### PR TITLE
Convert comments to tsdoc in REST sample

### PR DESCRIPTION
**Pull request by jt-nti**
Signed-off-by: James Taylor <jamest@uk.ibm.com>

### CODE REVIEW

**Review by mbwhite — DISMISSED** [review]

File: asset-transfer-basic/rest-api-typescript/src/errors.ts [hunk 1]
```diff
@@ -228,11 +232,12 @@ const matchTransactionDoesNotExistMessage = (
   return null;
 };
 
-/*
+/**
  * Handles errors from evaluating and submitting transactions.
  *
  * Smart contract errors from the the basic asset transfer samples do not use
```
Comment (lehors) [inline asset-transfer-basic/rest-api-typescript/src/errors.ts:238]:
```suggestion
 * Smart contract errors from the basic asset transfer samples do not use
```

**Review by lehors — CHANGES_REQUESTED** [review]
LGTM but I spotted a repeated word you might as well fix.

File: asset-transfer-basic/rest-api-typescript/src/errors.ts (hunk 1 above)
Comment (jt-nti) [reply asset-transfer-basic/rest-api-typescript/src/errors.ts]: Aha, thanks, fixed!

**Review by lehors — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**